src/player/player.js

```diff
@@ -80,6 +82,8 @@ export class Player extends Phaser.Sprite {
       this.body.moveUp(375);
       this.jumpTimer = this.game.time.now + 750;
     }
+
+    this.game.debug.geom(this.getBounds());
```

Comment:
I left it in do you could see the overlap of the sprites white you work on collision. But I probably should have removed it for the pr.